src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataServiceImpl.java

```diff
@@ -1377,14 +1362,12 @@ public void deleteFormDataByFormIdAndId(String formId, String id)
     }
 
     @Override
-    public AggregationResponse aggregateByFormIdFilterGroupBy(String formId, String filter, String groupBy, String operation)
-    {
+    public AggregationResponse aggregateByFormIdFilterGroupBy(String formId, String filter, String groupBy, String operation) throws JsonProcessingException {
         checkMongoCollectionIfExistsOrNot(formId);
-        Criteria criteria = new Criteria();
         List<AggregationOperation> aggregationOperationsList = new ArrayList<>();
         if(isNotEmpty(filter))
         {
-            createMultipleFilterCriteria(filter, criteria, aggregationOperationsList);
```

Comment:
why we need to change this?
You need to explain me this.

Reply:
I will explain